packages/tomls/src/omnibus/reya_network.toml

```diff
@@ -35,19 +35,19 @@ lotteryPotAccountId = "20092"
 rageTradeAccountId = "22262"
 
 [var.cannonClonePackages]
-coreRouterPackage = "reya-core:1.0.10@router"
+coreRouterPackage = "reya-core:1.0.11@router"
 coreAccountNftRouterPackage = "reya-core:1.0.1@account-nft-router"
 rUSDRouterPackage = "reya-rusd:1.0.0@router"
 tokensProxyPackage = "reya-tokens:1.0.0@proxy"
 tokensRouterPackage = "reya-tokens:1.0.0@router"
 passivePoolRouterPackage = "reya-exchange-passive-pool:1.0.4@router"
 passivePerpRouterPackage = "reya-instrument-passive-perp:1.0.16@router"
-peripheryRouterPackage = "reya-periphery:1.0.3@router"
+peripheryRouterPackage = "reya-periphery:1.0.4@router"
```

Comment:
need to sync this with @ab PR

Reply:
yes yes, he'll layer in his PR on top of mine